langchain/src/experimental/plan_and_execute/prompt.ts

```diff
@@ -31,3 +15,40 @@ Current objective: {current_step}
 {agent_scratchpad}
 
 You may extract and combine relevant data from your previous steps when responding to me.`;
+
+/**
+ * Add the tool descriptions to the planning system prompt in
+ * order to get a better suited plan that makes efficient use
+ * of the tools
+ * @param tools the tools available to the `planner`
+ * @returns
+ */
+export const getPlannerChatPrompt = (tools?: Tool[]) => {
+  const toolStrings = tools
+    ? tools.map((tool) => `${tool.name}: ${tool.description}`).join("\n")
+    : "";
+
+  const toolInstruction = tools
+    ? `You have a set of tools at your disposal to help you with this task: ${toolStrings}. You must consider these tools when coming up with your plan.`
+    : `Your steps should be general, and should not require a specific method to solve a step.`;
+
+    PLANNER_SYSTEM_PROMPT_MESSAGE_TEMPLATE = [
+    `Let's first understand the problem and devise a plan to solve the problem.`,
+    `Please output the plan starting with the header "Plan:"`,
+    `followed by a numbered list of steps.`,
+    `Please make the plan the minimum number of steps required`,
+    `to answer the query or complete the task accurately and precisely.`,
+    `${toolInstruction}`,
```

Comment:
We should make this always require tools to be passed in IMO

Reply:
Breaking changes are more ok here since it's under experimental

Reply:
Could you elaborate - I don't follow. thanks!

Reply:
Would we ever want to have a prompt where there are no tool instructions? Probably not right?

Reply:
Ah I see! Thank you. Yes you are absolutely right. I'll get the fixed pushed momentarily.